spartan/aztec-node/templates/statefulset.yaml

```diff
@@ -119,6 +121,11 @@ spec:
               {{- end }}
 
               start_cmd+=({{ join " " .Values.node.startCmd }})
+
+              {{- if .Values.node.preStartScript }}
+              {{ .Values.node.preStartScript | nindent 14 }}
+
+              {{- end }}
```

Comment:
My idea here is to use this script to run a DNS to discover validator pods and set the `PROVER_COORDINATION_URLS`